### PR TITLE
Add env vars for log configuration

### PR DESCRIPTION
**Pull request by slash4**

### CODE REVIEW

File: nginx.conf [hunk 1]
```diff
@@ -1,6 +1,6 @@
 user www-data;
 daemon off;
-worker_processes 4;
+worker_processes 8;
```
Comment (jasonwyatt) [inline nginx.conf]: Was there a specific need for this in relation to the logging changes?

Comment (slash4) [inline nginx.conf]: Absolutely not :) Sorry, it's a mistake.

Comment (jasonwyatt) [inline nginx.conf]: No problem, if you could roll that change back I'll go ahead and merge the PR.

**Comment by slash4** [comment]
Here it is !

**Comment by jasonwyatt** [comment]
Thanks!

**Pull request merged**